spec/calculate_spec.rb

```diff
@@ -1,9 +1,8 @@
-# require 'rspec/autorun'
 require 'rspec'
 require 'date'
 require './lib/calculate'
 
-describe Calculate do
+RSpec.describe Calculate do
   describe '::character_set' do
     it 'returns array of lowercase alphabet plus space' do
       expect(Calculate.character_set).to eql(["a", "b", "c", "d", "e", "f", "g", "h", "i", "j", "k", "l", "m", "n", "o", "p", "q", "r", "s", "t", "u", "v", "w", "x", "y", "z", " "])
```

Comment:
Ruby has ranges ["a".."z"].

Also, any reason we aren't also supporting Upper case and numbers (I didn't actually read the spec to check haha, so maybe that is the reason)?